docs/site/data/downloads/0-9-1.yml

```diff
@@ -30,3 +30,5 @@ source:
     file_name: v0.9.1.zip
     checksum: ''
     size: ''
+  - ova:
```

Comment:
This yaml is not formatted correctly, based on how you are trying to use it below. Should be the same as the `source` and `installer` tags.

Reply:
@stmcginnis please re-check.